stdlib/2and3/plistlib.pyi

```diff
@@ -5,8 +5,10 @@ from typing import (
     Type, TypeVar,
 )
 from typing import Dict as DictT
-from enum import Enum
 import sys
+if sys.version_info >= (3,):
+    from enum import Enum
+
```

Comment:
Move the definition of PlistFormat below into this "if" block, so you have all Enum related code in one block?

Reply:
Done.